packages/ia-components/sandbox/theatres/with-youtube-spotify/utils/flatten-album-data.test.js

```diff
@@ -17,32 +15,39 @@ describe('Music Player Util - Flatten Album Data', () => {
   } = dataFlattened;
   const [firstTrack] = tracks;
 
-  assert(dataFlattened instanceof Object);
+  expect(dataFlattened).toBeInstanceOf(Object);
+
+  describe('has main image file', () => {
+    test('when `*_itemimage.png` is available, it uses it as the image to display', () => {
+      expect(itemPhoto).toContain('_itemimage.png');
+    });
+  });
 
   describe('should always have certain keys at the top level', () => {
     test('has albumSpotifyYoutubeInfo', () => {
-      assert('albumSpotifyYoutubeInfo' in dataFlattened);
-      assert(albumSpotifyYoutubeInfo instanceof Object);
+      expect(dataFlattened).toHaveProperty('albumSpotifyYoutubeInfo');
+      expect(albumSpotifyYoutubeInfo).toBeInstanceOf(Object);
     });
     test('has tracks', () => {
-      assert('tracks' in dataFlattened);
-      assert(tracks instanceof Array);
+      expect(dataFlattened).toHaveProperty('tracks');
+      expect(tracks).toBeInstanceOf(Array);
     });
     test('has albumMetadaToDisplay', () => {
-      assert('albumMetadaToDisplay' in dataFlattened);
-      assert(albumMetadaToDisplay instanceof Object);
+      expect(dataFlattened).toHaveProperty('albumMetadaToDisplay');
+      expect(albumMetadaToDisplay).toBeInstanceOf(Object);
     });
     test('has identifier', () => {
-      assert('identifier' in dataFlattened);
-      assert(identifier instanceof Array);
+      expect(dataFlattened).toHaveProperty('identifier');
+      expect(identifier).toBeInstanceOf(Array);
     });
     test('has itemPhoto', () => {
-      assert('itemPhoto' in dataFlattened);
-      assert.equal(typeof itemPhoto, 'string');
+      expect(dataFlattened).toHaveProperty('itemPhoto');
+      expect(itemPhoto).toBeTruthy();
+      expect(itemPhoto).toBeDefined();
```

Comment:
**Suggestion** If the previous assertion that `itemPhoto` be truthy passes, then this will always pass. Perhaps reorder the two?

Reply:
oh interesting. i was unaware of that dependency. can do